saleor/graphql/account/mutations/permission_group.py

```diff
@@ -268,6 +275,38 @@ def clean_remove_users(cls, info, errors, cleaned_input):
             params = {"users": [graphene.Node.to_global_id("User", user.pk)]}
             cls.update_errors(errors, error_msg, "remove_users", code, params)
 
+    @classmethod
+    def check_if_users_can_be_removed(
+        cls, errors: dict, cleaned_input: dict, group: Group
+    ):
+        """Check if after removing users from group all permissions will be manageable.
+
+        After removing users from group, for each permission, there should be
+        at least one staff member who can manage it (has both “manage staff”
+        and this permission).
+        """
+        remove_users = cleaned_input["remove_users"]
+        add_users = cleaned_input.get("add_users")
+        manage_staff_permission = AccountPermissions.MANAGE_STAFF.value
+
+        # check if user with manage staff will be added to the group
+        if add_users:
+            if any([user.has_perm(manage_staff_permission) for user in add_users]):
+                return True
+
+        permissions = get_not_manageable_permissions_after_removing_users_from_group(
```

Comment:
I know naming gets quite long however maybe it's good to change it somehow (var name), below we check if there are "permissions" and if so, we raise an error :thinking:

Reply:
Yeah, I wanted to name it `not_manageable_permissions` but then the line was too long. After adding brackets around, it was removed by black, flake failed :) Of course, I can force commit to skip pre-commit checks, but I don't think is a good idea.